api/api/management/commands/documentmedia.py

```diff
@@ -293,6 +299,11 @@ def generate_notes(model: str, fields: list[FieldInfo]) -> tuple[str, set[str]]:
         if not field.is_relation and field.value_info.help_text:
             field_output += f"**Help text:** {field.value_info.help_text}\n\n"
             record = True
+        if field.is_relation and field.relation_info.doc:
+            field_output += (
+                f"**Model docstring:** {dedent(field.relation_info.doc)}\n\n"
```

Comment:
Would be nice to know which model this docstring is for without going back to the table to get that info.

```suggestion
                f"**`{field.relation_info.to}` docstring:** {dedent(field.relation_info.doc)}\n\n"
```

Reply:
Great idea, done in [d11bee1](https://github.com/WordPress/openverse/pull/4225/commits/d11bee165402e5da1d1cb85e8ee9b4d0e04f7dfa)!